package.json

```diff
@@ -42,6 +42,7 @@
     "piexifjs": "^1.0.4",
     "rate-limit-redis": "^1.7.0",
     "redis": "^3.1.2",
+    "sharp": "^0.28.3",
```

Comment:
I believe the CircleCI error is because `package-lock.json` was not committed. You should commit that file as well.